tests/integrations/config/test-custom.toml

```diff
@@ -105,9 +105,11 @@ enable-ttl = true
 ttl-check-poll-interval = "0s"
 txn-status-cache-capacity = 1000
 memory-quota = "123KB"
-max-ts-drift-allowance = "333s"
-max-ts-sync-interval = "44s"
-action-on-invalid-max-ts = "error"
+
+[storage.max-ts]
+max-drift = "333s"
+cache-sync-interval = "44s"
```

Comment:
This config is not used by `test_update_config`, it this expected?

Reply:
This file is used in test_serde_custom_tikv_config